packages/graph/hash_graph/lib/graph/src/types/mod.rs

```diff
@@ -16,17 +18,7 @@ impl AccountId {
     }
 }
 
-#[repr(transparent)]
-#[derive(Clone, Copy, Debug, sqlx::Type, PartialEq, Eq)]
-#[sqlx(transparent)]
-pub struct BaseId(Uuid);
-
-impl BaseId {
-    #[must_use]
-    pub const fn new(uuid: Uuid) -> Self {
-        Self(uuid)
-    }
-}
+pub type BaseId = Uri;
```

Comment:
I don't think it's worth the effort of undoing it on this PR, but I do wish we hadn't moved to the Uri as part of this PR because we're in a strange state where `VersionId` is a UUID, `BaseId` is a string, we don't really have proper mappings between anything and it's just inconsistent.

Let's aim to fix this in the next PR.